packages/core/src/sfpcommands/source/DeploySourceToOrgImpl.ts

```diff
@@ -215,72 +115,94 @@ export default class DeploySourceToOrgImpl implements DeploymentExecutor {
+            metdataDeployOptions.apiOptions.ignoreWarnings = true;
         }
         if (this.deployment_options['ignore_errors']) {
-            command += ` --ignoreerrors`;
+            metdataDeployOptions.apiOptions.rollbackOnError = false;
         }
 
-        return command;
+        return metdataDeployOptions;
     }
 
-    private convertApexTestSuiteToListOfApexClasses(apextestsuite: string): Promise<string> {
-        SFPLogger.log(
-            `Converting an apex test suite  ${apextestsuite} to its consituent apex test classes`,
-            null,
-            this.packageLogger
-        );
+    convertApexTestSuiteToListOfApexClasses(apexTestSuite: any) {
+        throw new Error('Method not implemented.');
+    }
 
-        let result = child_process.execSync(
-            `sfdx sfpowerkit:source:apextestsuite:convert  -n ${apextestsuite} --json`,
-            { cwd: this.project_directory, encoding: 'utf8' }
-        );
+    private async deploy(backingSourceDir: string, componentSet: ComponentSet) {
+        let deploymentOptions = await this.buildDeploymentOptions(backingSourceDir, this.target_org);
+        const deploy = await componentSet.deploy(deploymentOptions);
+
+        let startTime = Date.now();
+        SFPLogger.log(`Deploying to ${this.target_org} with id:${deploy.id}`, LoggerLevel.INFO, this.packageLogger);
+        // Attach a listener to check the deploy status on each poll
+        deploy.onUpdate((response) => {
+            const { status, numberComponentsDeployed, numberComponentsTotal } = response;
+            const progress = `${numberComponentsDeployed}/${numberComponentsTotal}`;
+            const message = `Status: ${status} Progress: ${progress}`;
+            SFPLogger.log(message, LoggerLevel.INFO, this.packageLogger);
+        });
 
-        let resultAsJSON = JSON.parse(result);
-        if (resultAsJSON['status'] == 0) {
-            return resultAsJSON['result'];
-        } else {
-            throw new Error(`Unable to convert apex test suite ${apextestsuite} ${resultAsJSON['message']}`);
-        }
+        deploy.onFinish((response) => {
+            let deploymentDuration = Date.now() - startTime;
+            const deploymentDurationAsDate = new Date(deploymentDuration);
+
+            if (response.response.success) {
+                SFPLogger.log(
+                    COLOR_SUCCESS(
+                        `Succesfully Deployed ${COLOR_HEADER(
+                            response.response.numberComponentsDeployed
+                        )} components in ${deploymentDurationAsDate.getMinutes()}:${deploymentDurationAsDate.getSeconds()} mins`
+                    ),
+                    LoggerLevel.INFO,
+                    this.packageLogger
+                );
+            } else
+                SFPLogger.log(
+                    COLOR_ERROR(
+                        `Failed to deploy after ${deploymentDurationAsDate.getMinutes()}:${deploymentDurationAsDate.getSeconds()} mins`
```

Comment:
Missing hours

Reply:
Not converted to hours.. do you think we want it?